### DISCUSSION TITLE
any way can specify airflow  using /bin/python3

### DISCUSSION THREAD

**Discussion by sinbargit** [apache/airflow#17403] [Q&A]
**Description**

any way can specify airflow  using /bin/python3

**Use case / motivation**

There is some important application using python2 in my server. So I can not uninstall python2 just using python3, and also can not link /bin/python2 to /bin/python3. But the airflow using /bin/python2 any way can specify airflow using /bin/python3?

Rather than telling us how you might implement this solution, try to take a
step back and describe what you are trying to achieve.

-->

**Are you willing to submit a PR?**

**Related Issues**

**Comment by potiuk** [comment]
`python3 -m airflow`

**Reply by sinbargit** [reply]
There is an install script in https://airflow.apache.org/docs/apache-airflow/stable/tutorial.html. How can I change to python3 with it

**Reply by potiuk** [reply]
Again use `python3 -m airflow` instead of `airflow` . You can also add a bash script to run your command and make python in the right path. Or use pyenv + pyenv virtualenv https://github.com/pyenv/pyenv-virtualenv to setup virtualenv and swtch to it by default.

Please read the docs about virtualenv in python https://docs.python.org/3/library/venv.html - this is pretty basic python knowledge and nothing specific to airflow.
